spring-cloud-gcp-samples/spring-cloud-gcp-logging-sample/src/test/java/com.example/LoggingSampleApplicationTests.java

```diff
@@ -97,7 +97,7 @@ public void testLogRecordedInStackDriver() {
 
 		String logFilter = String.format(LOG_FILTER_FORMAT, traceHeader);
 
-		await().atMost(60, TimeUnit.SECONDS)
+		Awaitility.await().atMost(60, TimeUnit.SECONDS)
```

Comment:
Let's ask Spring Cloud guys to include this one into exclude as well!
In that https://github.com/spring-cloud/spring-cloud-build/issues project.